test_tf.py

```diff
@@ -0,0 +1,153 @@
+import os
+import subprocess
+import unittest
+from main import get_policy_check_type, get_required_inputs, get_optional_inputs, get_sub_command
+from main import get_treat_findings_as_non_blocking_flag, build_command, execute_command, set_output
+
+CLI_POLICY_VALIDATOR = "tf-policy-validator"
+
+POLICY_CHECK_TYPE = "INPUT_POLICY-CHECK-TYPE"
+
+NOT_VALIDATE_POLICY = "NOT_VALIDATE_POLICY"
+VALIDATE_POLICY = "VALIDATE_POLICY"
+CHECK_NO_NEW_ACCESS = "CHECK_NO_NEW_ACCESS"
+CHECK_ACCESS_NOT_GRANTED = "CHECK_ACCESS_NOT_GRANTED"
+
+COMMON_REQUIRED_INPUTS = {"INPUT_TEMPLATE-PATH", "INPUT_REGION"}
+
+TREAT_FINDINGS_AS_NON_BLOCKING = "INPUT_TREAT-FINDINGS-AS-NON-BLOCKING"
+
+# assertRaises(SomeCoolException, mymod.myfunc)
+class TfpvTest(unittest.TestCase):
+    # test_get_type_case1: failure expected because required os.environ[] are not set")
+    @unittest.expectedFailure
+    def test_get_type_case1(self):
+        assertRaises(ValueError, get_policy_check_type)
+
+    # test_get_type_case2: failure expected because os.environ[] is set to an invalid value")
+    @unittest.expectedFailure
+    def test_get_type_case2(self):
+        os.environ[POLICY_CHECK_TYPE] = NOT_VALIDATE_POLICY
+        assertRaises(ValueError, get_policy_check_type)
+
+    def test_get_type_case3(self):
```

Comment:
You should probably run this check for all the possible valid values for this input. 

You can leverage the parameterized tests to use the same test method, various inputs. Blog explaining that - https://medium.com/@samarthgvasist/parameterized-unit-testing-in-python-9be82fa7e17f